### PR TITLE
add link to configuration profiles

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9758

### CODE REVIEW

File: Teams/devices/check-in-and-room-release.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ This feature can be used in a standalone Teams panel deployment. You can also pa
 
 The shared mailbox associated with the Teams panel needs to have the correct time zone set for this feature to work correctly. For information about how to set the time zone for shared mailboxes, see [Time zone settings for shared mailboxes in Outlook on the web](/exchange/troubleshoot/outlook-on-the-web-issues/shared-mailboxes-time-zone-setting).
 
+You can also use configuration profiles to apply this feature to a set of devices, for more information, see [Use configuration profiles in Teams](device-management.md#use-configuration-profiles-in-teams).
```
Comment (JohanFreelancer9) [inline Teams/devices/check-in-and-room-release.md:34]:
```suggestion
You can also use configuration profiles to apply this feature to a set of devices. For more information, see [Use configuration profiles in Teams](device-management.md#use-configuration-profiles-in-teams).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Approved with a minor change. Thanks.

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@dstrome

**Pull request merged**